### DISCUSSION TITLE
Profile Banner Wrong Size?

### DISCUSSION THREAD

**Discussion by WalmartWhiteCat** [ppy/osu#30898] [Ideas] [answered]
I don't know if anyone has mentioned this yet (I tried searching for banners and saw nothing related), but banners on Lazer profiles are the wrong size. On the website, my banner fits nicely, but on Lazer it's cut off a bit.

Website:
![image](https://github.com/user-attachments/assets/7f423a23-3fec-4136-b899-8e2b00a3e2da)

Lazer:
![image](https://github.com/user-attachments/assets/1c5ee724-46b8-42ff-b9c9-5f96c80fc38c)

**Comment by Joehuu** [comment, marked as answer]
There's an explanation of why it's like that in https://github.com/ppy/osu/issues/30298.

**Reply by WalmartWhiteCat** [reply]
Oh, thank you

**Reply by WalmartWhiteCat** [reply]
idk why I can never find posts when I search for things I wanna make discussions about. It's like a curse or smth I swear
